app/workers/nexaas/async/collector/async_resource_job.rb

```diff
@@ -20,6 +22,28 @@ def call_for_method(klass_name, klass_method, args=[])
             klass_name.to_s.constantize.send(klass_method)
           end
         end
+
+        def instrument_start(collect_id, user_id, klass_name, klass_method)
+          @start = Time.current.to_i
+          ActiveSupport::Notifications.instrument("#{instrumentation_context}.start", {
+            collect_id: collect_id, user_id: user_id,
+            class_name: klass_name, class_method: klass_method,
+            start: @start
+          })
+        end
+
+        def instrument_finish(collect_id, user_id, klass_name, klass_method)
+          duration = (Time.current.to_i - @start)
+          ActiveSupport::Notifications.instrument("#{instrumentation_context}.finish", {
+            collect_id: collect_id, user_id: user_id,
+            class_name: klass_name, class_method: klass_method,
+            duration: duration
```

Comment:
should also supply a `finish` key with the time in integers, like we do on the "start"